### PR TITLE
Update Plate 4 normalization to standardize to both WT and Null controls

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Update Plate 4 normalization to standardize to both WT and Null controls

In this small PR, Plate 4 normalization was updated where the `samples` parameter now includes WT genotype and Null genotype at the 0 dose for siRNA. Processing steps were reran for Plate 4 only and added into the repo.

### CODE REVIEW

File: 3.processing_features/scripts/2.pycytominer_singlecell_pipelines.py [hunk 1]
```diff
@@ -143,9 +143,10 @@
     # set default for samples to use in normalization
     samples = "all"
 
-    # Only for Plate 4, we want to normalize to no siRNA treatment Null cells (controls)
+    # Only for Plate 4, we want to normalize to no siRNA treatment Null and WT cells (controls)
     if plate == "Plate_4":
-        samples = "Metadata_Concentration == 0.0 and Metadata_genotype == 'Null'"
+        samples = "Metadata_Concentration == 0.0 and (Metadata_genotype == 'Null' or Metadata_genotype == 'WT')"
```
Comment (MattsonCam) [inline 3.processing_features/scripts/2.pycytominer_singlecell_pipelines.py:148]: This should work!

**Review by MattsonCam — APPROVED** [review]
LGTM @jenna-tomkinson, good job!

File: 3.processing_features/scripts/2.pycytominer_singlecell_pipelines.py (hunk 1 above)
Comment (jenna-tomkinson) [reply 3.processing_features/scripts/2.pycytominer_singlecell_pipelines.py:148]: I think so too! 😄

**Comment by jenna-tomkinson** [comment]
@MattsonCam Thank you for the review! I will be merging now! Exicted to see what is to come 😄

**Pull request merged**